backend/src/main/java/com/bakdata/conquery/apiv1/forms/export_form/ExportForm.java

```diff
@@ -94,15 +96,20 @@ public void resolve(QueryResolveContext context) {
 		timeMode.resolve(context);
 		prerequisite = queryGroup.getQuery();
 
+		List<Resolution> resolutionsFlat = resolution.stream()
+													 .flatMap(ResolutionShortNames::correspondingResolutions)
+													 .distinct()
+													 .toList();
```

Comment:
ich glaube set wäre hier sogar sinnvoller, du hast es ja eh schon distinct

Reply:
Es beeinflusst die Reihenfolge der DateContexts und somit die Sortierung der Zeilen in einem Export Formular ;)

Reply:
Es soll immer grob nach fein sein